pubsub/redis/metadata.go

```diff
@@ -14,6 +14,17 @@ type metadata struct {
 	host string
 	// The Redis password
 	password string
+	// Dial timeout for establishing new connections.
+	dialTimeoutInSec int
```

Comment:
For consistency, this should be parsed as a `time.Duration` like `redeliverInterval` and `processingTimeout` are and the `InSec` suffix removed.

Reply:
I will add more  connection settings, and keep their format identical to definition in redis client config as much as possible.